### DISCUSSION TITLE
Ant Design官网国内镜像站点路由错误

### DISCUSSION THREAD

**Discussion by mayef** [ant-design/ant-design#45407] [General]
重现链接：
https://4x-ant-design.antgroup.com/components/button-cn

重现步骤：
1. 打开https://ant-design.antgroup.com/components/button-cn
2. 切换4.x版本

期望结果：
4.x版本文档

实际结果：
404

补充说明：
正确链接：https://4x-ant-design.antgroup.com/components/button-cn/
不止Button组件，其他组件如Divider等也可能存在同样问题

**Comment by zombieJ** [comment]
托管的服务器不一样，route 规则也不同。印象里不太好解

**Reply by yoyo837** [reply]
切换major版本直接跳首页得了
